gen/suites/vm_violations/main.go

```diff
@@ -50,4 +54,62 @@ func main() {
 			Func:     callerValidation(&chaos.CallerValidationBranchTypeNilSet, exitcode.SysErrForbidden),
 		},
 	)
+
+	// Build an unknown Actor CID.
+	unknownCid, err := cid.V1Builder{Codec: cid.Raw, MhType: multihash.IDENTITY}.Sum([]byte("fil/1/unknown"))
+	if err != nil {
+		panic(err)
+	}
+
+	bobAddr := func(v *Builder) address.Address { return v.Actors.Handles()[1].Robust }
+	goodAddr := func(v *Builder) address.Address { return v.Actors.Handles()[1].NextActorAddress(100, 0) }
+	undefAddr := func(v *Builder) address.Address { return address.Undef }
+
+	g.MessageVectorGroup("actor_creation",
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "fails-with-existing-address",
+				Version: "v1",
+				Desc:    "verifies that CreateActor aborts when provided an existing address",
+			},
+			Selector: "chaos_actor=true",
+			Func:     createActor(bobAddr, builtin.AccountActorCodeID),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "fails-with-undef-addr",
+				Version: "v1",
+				Desc:    "verifies that CreateActor aborts when provided an address.Undef",
+			},
+			Selector: "chaos_actor=true",
+			Func:     createActor(undefAddr, builtin.AccountActorCodeID),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "fails-with-unknown-actor-cid",
+				Version: "v1",
+				Desc:    "verifies that CreateActor aborts when provided an unknown actor code CID",
+			},
+			Selector: "chaos_actor=true",
+			Func:     createActor(goodAddr, unknownCid),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "fails-with-unknown-actor-cid-undef-addr",
+				Version: "v1",
+				Desc:    "verifies that CreateActor aborts when provided an unknown actor code CID and an undef address",
+			},
+			Selector: "chaos_actor=true",
+			Func:     createActor(undefAddr, unknownCid),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "fails-with-undef-actor-cid-undef-addr",
+				Version: "v1",
+				Desc:    "verifies that CreateActor aborts when provided an undef actor code CID and an undef address",
+			},
+			Selector: "chaos_actor=true",
+			Func:     createActor(undefAddr, cid.Undef),
+		},
+	)
```

Comment:
Missing a case for undef CID with valid address.

Reply:
Good call; added: https://github.com/filecoin-project/test-vectors/pull/41/commits/31882d52a71cb9ede9bab15409766c82dd93ce74.